lib/livebook_web/live/home_live.ex

```diff
@@ -38,8 +38,23 @@ defmodule LivebookWeb.HomeLive do
   def render(assigns) do
     ~H"""
     <div class="flex grow h-full">
+      <div id="show-sidebar-tooltip" class="hidden fixed top-[1rem] left-[0.5rem]">
+        <span class="tooltip right distant" data-tooltip="show sidebar">
+          <button class="text-2xl text-gray-400 hover:text-gray-600 focus:text-gray-50 rounded-xl h-10 w-10 flex items-center justify-center"
+            aria-label="show sidebar"
+            phx-click={show_sidebar()}>
+            <.remix_icon icon="menu-line" />
+          </button>
+        </span>
+      </div>
```

Comment:
This can also go in `SidebarHelpers` and be shared, probably same with the button item below :)